### ISSUE TITLE
Debts datatable styles

### ISSUE THREAD

**Issue by rfultz** [fecgov/fec-cms#4414]
@rfultz commented on [Mon Feb 22 2021](https://github.com/fecgov/openFEC/issues/4778)

After https://github.com/fecgov/openFEC/issues/4758 is done, we'll need to determine the look / organization of the table / how the data will be presented to the viewers

Blocked by https://github.com/fecgov/openFEC/issues/4758

Blocking https://github.com/fecgov/fec-cms/issues/4413

**Comment by JonellaCulmer** [comment]
Closing in favor of https://github.com/fecgov/fec-cms/issues/4413
